MailCore/Models/Folder.swift

```diff
@@ -301,6 +301,15 @@ public class Folder: Object, Codable, Comparable, Identifiable {
         }
     }
 
+    public var isDroppable: Bool {
+        switch role {
+        case .scheduledDrafts, .draft, .snoozed, .sent:
+            return false
+        default:
+            return true
+        }
+    }
```

Comment:
We need a more generic name and reuse the condition here -> MoveEmailView.swift:51